Eshop/apps/galleries/models.py

```diff
@@ -3,7 +3,10 @@
 from django.conf import settings
 from django.db import models
 
+from apps_generic.whodidit.models import WhoDidIt
 
+
+# class Gallery(WhoDidIt):
```

Comment:
ValueError at /galleries/
Cannot assign "<django.contrib.auth.models.AnonymousUser object at 0x04744C58>": "Gallery.created_by" must be a "User" instance.
Request Method: POST

...\feb_py_eshop\Eshop\apps_generic\whodidit\models.py in set_who_did_it
44                 instance.created_by = user 

...\feb_py_eshop\Eshop\apps\galleries\models.py in save
57   super().save(*args, **kwargs)